packages/core/src/button/base-button.element.scss

```diff
@@ -41,7 +41,7 @@
   text-align: center;
   text-decoration: var(--text-decoration);
   text-overflow: ellipsis;
-  transform: translateZ(0px); // used for ::after hover
+  transform: scale(1);
```

Comment:
I think the `translateZ()` that was previously here was a legacy style that is no longer used. In clarity UI, we used to use translate to push the button down one pixel when it was clicked. We are no longer doing that.

I tried toggling this translate on and off to see if it made any difference. I didn't see any change.